src/App.js

```diff
@@ -24,7 +24,7 @@ class App extends Component {
     const { history } = this.props;
     insights.chrome.init();
     insights.chrome.auth.getUser().then((user) => this.setState({ userReady: true, isAdmin: user.identity.user.is_org_admin }));
-    insights.chrome.identifyApp('rbac');
+    insights.chrome.getApp() === 'rbac' ? insights.chrome.identifyApp('rbac') : insights.chrome.identifyApp('my-user-access');
```

Comment:
Can we have something like: `insights.chrome.identifyApp(insights.chrome.getApp())`?